client/js/widgets/button.js

```diff
@@ -82,23 +89,31 @@ class Button extends Widget {
         if(isValidCollection(a.collection))
           for(const w of collections[a.collection])
             w.click();
+        else
```

Comment:
These SHOULD be part of isValidCollection and isValidID. Doesn't the push in there work? I might not have tested it.

Reply:
You're right. It does do the logging. I missed that. Removed here & in a couple other spots I had added it.

I added a corresponding test for the case where the collection hasn't been created by a prior function.